.github/workflows/integration-tests.yml

```diff
@@ -20,7 +20,9 @@ jobs:
       - uses: DeterminateSystems/nix-installer-action@main
       - uses: DeterminateSystems/magic-nix-cache-action@main
       - name: Install neovim
-        uses: rhysd/action-setup-vim@v1 # Used by the 'run' integration test
+        # Used by the 'run' integration test
+        # Custom workflow used, see: https://github.com/rhysd/action-setup-vim/pull/40
+        uses: danarnold/action-setup-vim@linux-releases
```

Comment:
Looks like it's still failing. Let's just pin the Neovim version to 0.10.13 until they fix the github action.

Reply:
(I've pinned the nvim version in #357).

Reply:
Alrighty, will close this then.